tests/models/test_DurableOrchestrationContext.py

```diff
@@ -100,3 +100,19 @@ def test_get_input_json_str():
     result = context.get_input()
 
     assert 'Seattle' == result['city']
+
+def test_version_equals_version_from_first_execution_started_event():
+    builder = ContextBuilder('test_function_context')
+    builder.history_events = []
+    builder.add_orchestrator_started_event()
+    builder.add_execution_started_event(name="TestOrchestrator", version="1.0")
+    builder.add_execution_started_event(name="TestOrchestrator", version="2.0")
```

Comment:
It's not valid to have two execution_started events in the same orchestration history, so this test doesn't really make sense. Remove the "2.0" execution started event and update the name of the test accordingly.